ext/sockets/sockets.c

```diff
@@ -1580,7 +1580,8 @@ PHP_FUNCTION(socket_recvfrom)
 
 			if (arg6 == NULL) {
 				zend_string_efree(recv_buf);
-				WRONG_PARAM_COUNT;
+				zend_wrong_param_count();
+				RETURN_THROWS();
```

Comment:
This feels like it should be a better error message, as it seems for IP address a 6th argument is required?